release.php

```diff
@@ -47,29 +48,30 @@
 `git fetch;`;
 $result = `git tag;`;
 $tag = '0.0.1';
-if($result !== null) {
+if ($result !== null) {
     $tags = [];
 
     foreach (explode(PHP_EOL, $result) as $line) {
         $matches = [];
         preg_match('/[0-9]{1,}.[0-9]{1,}.[0-9]{1,}/', $line, $matches);
 
-        if(empty($matches)) {
+        if (empty($matches)) {
             continue;
         }
 
         $explodedTag = explode('.', $matches[0]);
-        $tags[$explodedTag[0]][$explodedTag[1]][$explodedTag[2]] = $matches[0];
+        $tagNumber = $explodedTag[0] . '.' . $explodedTag[1] . '.' . $explodedTag[2];
+
+        $tagsArray = array_push($tags, $tagNumber);
```

Comment:
$tagsArray = not needed because it is never used.
BTW: array_push. What is this method returning?